src/components/FilterBar/__tests__/FilterBadge.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import configureMockStore from 'redux-mock-store'
+import FilterBadge from '../FilterBadge'
+
+const mockStore = configureMockStore()
+
+test('Filter Badge displays badge containing number of items in filter', () => {
+    const filter = {
+        id: 'ponies',
+        name: 'Ponies',
+        values: [{ name: 'Rainbow Dash' }, { name: 'Twilight Sparkle' }],
```

Comment:
Now I finally know who added all of the my little pony names to DV 🐴 😄

Reply:
🦄